install.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+import tkinter as tk
+import lsb_release
+import subprocess
+import webbrowser
+
+release = lsb_release.get_distro_information()
+docs = 'https://paparazzi-uav.readthedocs.io'
+
+window = tk.Tk()
+window.title("Install Paparazzi-UAV")
+window.minsize(300,600)
+
+text = tk.Label(window, text="OS:" + release['ID'] + ' ' + release['RELEASE'])
+text.pack()
+
+
+# Logo
+image = tk.PhotoImage(file="./data/pictures/Penguin.gif")
+img = tk.Label(window, image=image)
+img.pack()
+
+
+def execute(cmd):
+    print(cmd)
+    p = subprocess.Popen(cmd, shell=True)
+    p.communicate() # wait until ready
+    print('=================================================')
+    print(cmd)
+    print('READY (verify above)')
+    print('Check documentation to resolve problems:', docs)
+    print('=================================================')
+
+def view(link):
+    webbrowser.open(link)
+
+def cmd_repo():
+    execute('sudo add-apt-repository -y ppa:paparazzi-uav/ppa && sudo apt-get update')
```

Comment:
How do you enter the sudo password ? In the terminal running the program ?

Reply:
This could be done with `pkexec`

Reply:
When running the ./install.sh script, it asks for a password and then all install clicks are executed without further prompts. If you have a user-prompt (e.g. yes/no, pw, etc, ... ) you can type it in the console that started the script.

Reply:
I didn't use `pkexec` because then I got into trouble with the $DISPLAY environment.

Reply:
I just checked again: it says: 774kb of memory will be used for installing python-tk